common/core/desktop/include/keyman/keyboardprocessor_vkeys.h

```diff
@@ -19,27 +19,29 @@ enum km_kbp_modifier_state {
   KM_KBP_MODIFIER_RALT        = 1 << 3,
   KM_KBP_MODIFIER_SHIFT       = 1 << 4,
   KM_KBP_MODIFIER_CTRL        = 1 << 5,
-  KM_KBP_MODIFIER_ALT         = 1 << 6
+  KM_KBP_MODIFIER_ALT         = 1 << 6,
   /*
-    KM_KBP_MODIFIER_CAPS        = 1 << 7,
-    KM_KBP_MODIFIER_NOCAPS      = 1 << 8,
-    KM_KBP_MODIFIER_NUMLOCK     = 1 << 9,
-    KM_KBP_MODIFIER_NONUMLOCK   = 1 << 10,
-    KM_KBP_MODIFIER_SCROLLOCK   = 1 << 11,
-    KM_KBP_MODIFIER_NOSCROLLOCK = 1 << 12,
-    KM_KBP_MODIFIER_VIRTUALKEY  = 1 << 13,
+    KM_KBP_MODIFIER_META        = 1 << 7,
```

Comment:
Should this be `KM_KBP_MODIFIER_META` or should this be something like `RESERVED`? Probably doesn't matter too much since it's a comment...

Reply:
```suggestion
    KM_KBP_MODIFIER_META        = 1 << 7,    // Either Meta-key flag (tentative).  Not usable by keyboards currently
                                             // Used internally (currently, only by KMW) to ensure Meta-key 
                                             // shortcuts safely bypass rules
                                             // Meta key = Command key on macOS, Windows key on Windows    
```